helm/defectdojo/values.yaml

```diff
@@ -149,7 +149,7 @@ django:
     app_settings:
       processes: 2
       threads: 2
-    enable_ptvsd: false  # this also requires DD_DEBUG to be set to True
+    enable_debugpy: false  # this also requires DD_DEBUG to be set to True
```

Comment:
Could you name it just `enable_debug` or something like that so we don't have to rename everything every time we switch from one lib to another?

Reply:
Let me change that quickly